compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/PipelineTranslationContext.java

```diff
@@ -275,7 +275,7 @@ DAGBuilder getBuilder() {
     if (srcTransform instanceof FlattenTransform) {
       return CommunicationPatternProperty.Value.ONE_TO_ONE;
     }
-    if (dstTransform instanceof GBKTransform
+    if ((dstTransform instanceof GBKTransform && !((GBKTransform) dstTransform).getIsPartialCombining())
```

Comment:
Could you please add some comments here?

Reply:
Yes, I added them. Thank you.